server/cron.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/robfig/cron"
+)
+
+// Cron manage all cron jobs of this plugin
+// behind the scene it's a facade to github.com/robfig/cron
+type Cron struct {
+	p *Plugin
+	c *cron.Cron
+}
+
+// NewCron return a cron
+func NewCron(p *Plugin) (*Cron, error) {
+	c := cron.New()
+	config := p.getConfiguration()
+
+	if err := c.AddFunc("@every 1h", func() {
```

Comment:
This is the part i'm afraid of : evry hour for all subscriptions check if a web hook is running, if not create one... I can improve it by adding info in KVStore and just ask for organization with new project. But if a user has huge subscriptions it will be time consuming and potentially generate lot of traffic...